spec/models/rental_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Rental, type: :model do
+  let!(:user) { User.create(username: 'robbie', password: 'password') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.